### PR TITLE
feat(e2e): add local host support bundle test

### PR DESCRIPTION
**Pull request by DexterYan**
## Description, Motivation and Context

- add local host support bundle test
- add preflight e2e go test 

## Checklist

- [x] New and existing tests pass locally with introduced changes.
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] The commit message(s) are informative and highlight any breaking changes
- [x] Any documentation required has been added/updated. For changes to https://troubleshoot.sh/ create a PR [here](https://github.com/replicatedhq/troubleshoot.sh/pulls)

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: test/e2e/support-bundle/host_local_collector_e2e_test.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+			cmd.Stdout = &out
+			err := cmd.Run()
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			defer func() {
+				err := os.Remove(fmt.Sprintf("%s.tar.gz", supportbundleName))
+				if err != nil {
+					t.Fatal("Error remove file:", err)
```
Comment (nvanthao) [inline test/e2e/support-bundle/host_local_collector_e2e_test.go]: prefer `t.Error` over `t.Fatal` to allow tests to continue running.
ref: https://go.dev/wiki/TestComments#keep-going

**Review by nvanthao — CHANGES_REQUESTED** [review]
LGTM, just 1 nit

File: test/e2e/support-bundle/host_local_collector_e2e_test.go (hunk 1 above)
Comment (DexterYan) [reply test/e2e/support-bundle/host_local_collector_e2e_test.go]: thank you! I have changed the PR accordingly.

**Review by nvanthao — APPROVED** [review]
LGTM

**Pull request merged**